backend/token.json

```diff
@@ -0,0 +1 @@
+{"access_token":"ya29.Il-iB2U2iJW3o32UtflgV2chshD21lNoZg6MJ-jg5T5AySb2gGlDXESmQDJ8CndqBwyXc_PnU0wIvNw6AIen8rKC-pNT3YqhyZtZADshIdLOm0VR9ykHaAl8mLbn4R-jSA","refresh_token":"1//0695hzuBEp1rMCgYIARAAGAYSNwF-L9IrJZb4tWBLdlFguzGIkYmU70HKdDD_VHdg1iA9e544kHPr__lj81KEdZZm7iD7xcgKIZY","scope":"https://www.googleapis.com/auth/spreadsheets.readonly","token_type":"Bearer","expiry_date":1571881288026}
```

Comment:
Merge master into your branch to resolve merge conflicts and also get updated version of .gitignore so this file isn't committed. May have to follow this: https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore